src/main/java/io/jenkins/docker/connector/DockerComputerSSHConnector.java

```diff
@@ -109,7 +109,7 @@ public void setJvmOptions(String jvmOptions) {
     public String getJavaPath() {
         return javaPath;
     }
-
+ 
```

Comment:
You've added a trailing whitespace to this line.
Please remove the whitespace and put the line back how it was.